### PR TITLE
My example

### PR DESCRIPTION
**Pull request by mw-hrastega**
New example on how to use the Jenkins plugin for MATLAB to run a set of MATLAB tests.

### CODE REVIEW

File: Examples/Run-MATLAB-Tests.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Run MATLAB Tests on Jenkins Server
```
Comment (mcafaro) [inline Examples/Run-MATLAB-Tests.md]: Can the `Examples` folder be changed to something like `doc`? Or at least `examples`, lowercase. That would align more with the folder naming convention of Maven projects.

Comment (nbhoski) [reply Examples/Run-MATLAB-Tests.md]: +1 on changing the folder name to examples

File: src/examples/Run-MATLAB-Tests.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+2. In the configuration window of your Jenkins project, navigate to the **Source Code Management** section and click **Git**. This option enables Jenkins to interface with a remote repository.
+3. Navigate to the main page of the GitHub repository that hosts your tests. Click **Clone or download** and copy the web URL to your clipboard.
+4. In the Jenkins interface, paste the URL into the **Repository URL** field of the **Source Code Management** section. (You also can specify the branch to build in the **Branch Specifier** field.)
+5. In the **Build** section of Jenkins, click **Add build step > Run MATLAB Tests**. A new window opens that enables you to configure the Jenkins plugin for MATLAB. Paste the full path to the MATLAB root folder into the **MATLAB root** field; choose the default Automatic test mode from the **Test mode** drop-down list; and select your desired test artifacts to be generated in the project wrokspace. (The build in this example is configured to generate Cobertura code coverage and JUnit test result reports.) For more information on how to configure the Jenkins plugin for MATLAB, see [Jenkins MATLAB Plugin](https://github.com/jenkinsci/matlab-plugin).
```
Comment (acampbel) [inline src/examples/Run-MATLAB-Tests.md]: typo "wrokspace"

File: src/examples/Run-MATLAB-Tests.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+![configure_plugin](https://user-images.githubusercontent.com/48831250/71114689-5900de80-219e-11ea-8e1c-4f6914bdf893.png)
```
Comment (acampbel) [inline src/examples/Run-MATLAB-Tests.md]: Need zn updated screenshot

File: src/examples/Run-MATLAB-Tests.md (hunk 2 above)
Comment (mw-hrastega) [reply src/examples/Run-MATLAB-Tests.md]: Thanks for catching it. Fixed.

File: Examples/Run-MATLAB-Tests.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Prerequisites
+Running the tests in this example requires you to interface Jenkins with MATLAB as well as a remote repository.
+
+* To run MATLAB tests and generate test artifacts, you must install the Jenkins plugin for MATLAB. For information on how to install a plugin in Jenkins, see [Managing Plugins](https://jenkins.io/doc/book/managing/plugins/).
+* Jenkins runs the tests in a branch of the remote repository that is specified by a URL. To follow the build configuration steps, you must upload the *Times Table App* example project to a remote GitHub&reg; repository.
+* To publish the test artifacts using post-build actions, relevant plugins must be installed on Jenkins. For example, the *Cobertura Plugin* integrates Cobertura coverage reports to Jenkins, and *JUnit plugin* enables JUnit-format test results to be published. 
+
+## Configure Jenkins to Run MATLAB Tests
+Configure the Jenkins build by following these steps:
```
[the remark this answers is not in the document]
Comment (mw-hrastega) [reply Examples/Run-MATLAB-Tests.md]: I modified the text as below. Please let me know if you have any flags.

## Configure Jenkins Project to Run MATLAB Tests
Create a new project and configure it by following these steps:

File: Examples/Run-MATLAB-Tests.md (hunk 1 above)
Comment (mw-hrastega) [reply Examples/Run-MATLAB-Tests.md]: I renamed it to examples and placed it under src. This is consistent with the MathWorks doc folder structure (matlab/doc).

File: Examples/Run-MATLAB-Tests.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+5. In the **Build** section of Jenkins, click **Add build step > Run MATLAB Tests**. A new window opens that enables you to configure the Jenkins plugin for MATLAB. Paste the full path to the MATLAB root folder into the **MATLAB root** field; choose the default Automatic test mode from the **Test mode** drop-down list; and select your desired test artifacts to be generated in the project wrokspace. (The build in this example is configured to generate Cobertura code coverage and JUnit test result reports.) For more information on how to configure the Jenkins plugin for MATLAB, see [Jenkins MATLAB Plugin](https://github.com/jenkinsci/matlab-plugin).
+
+![configure_plugin](https://user-images.githubusercontent.com/48831250/71114689-5900de80-219e-11ea-8e1c-4f6914bdf893.png)
+
+6. In the **Post-build Actions** section of Jenkins, add two post-build actions to publish the Cobertura code coverage and JUnit test result reports that the build generates. For each report, provide the path to the report file. Jenkins stores the artifacts in the *matlabTestArtifacts* folder of the workspace.
```
[the remark this answers is not in the document]
Comment (mw-hrastega) [reply Examples/Run-MATLAB-Tests.md]: I have mentioned the need for these plugins in the "prerequisites" section. Please see the third bullet. I included links to the two plugins there. Please let me know if this makes sense. Do the links land on right pages?

File: TimesTableApp/TimesTableApp.prj [hunk 6]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MATLABProject xmlns="http://www.mathworks.com/MATLABProjectFile" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0"/>
```
[the remark this answers is not in the document]
Comment (mw-hrastega) [reply TimesTableApp/TimesTableApp.prj]: I rephrased the text. Please let me know what you think.

File: src/examples/Run-MATLAB-Tests.md (hunk 2 above)
Comment (mw-hrastega) [reply src/examples/Run-MATLAB-Tests.md]: Fixed

File: src/examples/Run-MATLAB-Tests.md (hunk 3 above)
Comment (mw-hrastega) [reply src/examples/Run-MATLAB-Tests.md]: Since we don't have the new UI released, I can't update the screenshot now. I will ask Nikhil for help.

**Pull request closed without merging**